instructions/docs/add-the-encryption-sdk.md

```diff
@@ -106,20 +106,20 @@ const Body = fileStream.pipe(encryptStream(encryptKeyring));
 
```

Comment:
Should the javascript also say  Find the store function and edit it...

Reply:
Hmm. Thoughts, @seebees ?